packages/wrangler/src/__tests__/deploy.test.ts

```diff
@@ -10069,7 +10069,7 @@ export default{
 			fs.writeFileSync(
 				"index.js",
 				`
-      import path from 'path';
+      import path from 'node:path';
```

Comment:
This change is not required - the output is the same with or without the prefix.
Added only for consistency with l10093 below